public/admin/index.html

```
@@ -1191,6 +1203,122 @@ <h4>Broadcast</h4>
+                <input type="hidden" id="challengeAdminId" />
+                <label class="form-inline">
+                  <span>Name</span>
+                  <input type="text" id="challengeAdminName" maxlength="80" required />
+                </label>
+                <label class="form-inline">
+                  <span>Language</span>
+                  <input type="text" id="challengeAdminLang" pattern="[a-z]{2}(-[A-Z]{2})?" required maxlength="6" value="en" />
+                </label>
+                <label class="form-inline">
+                  <span>Word length</span>
+                  <input type="number" id="challengeAdminWordLength" min="3" max="12" value="5" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)